### DISCUSSION TITLE
Logging in as the admin.

### DISCUSSION THREAD

**Discussion by brajmohanT** [arlbibek/dVoting#4] [Q&A] [answered]
**How Could I Run this app as admin?**

I run the app and it is running as the voter on my end, how could I run as the admin?

**Comment by brajmohanT** [comment, marked as answer]
Okay I got it.🙏

**Reply by rakshitha-2710** [reply]
Sir please say how to run the admin page

**Reply by brajmohanT** [reply]
connect your wallet with the first account that is provided by the ganache/truffle suite.
by default, the first pseudo account is the admin and the other 9 are for voters.

**Reply by yashkusumkar02** [reply]
connected my wallet with the first account that is provided by the ganache/truffle suite. but it is not showing 

https://github.com/arlbibek/dVoting/assets/87336351/ae172575-f578-4a5a-991e-fe736ecb5ade

**Comment by arlbibek** [comment]
By default, the one who deploys the smart contract to the network is the administrator (usually its account on index 0 of Ganache), you will need to import that account using private key in your blockchain wallet (i.e. Metmask) and switch to that account in order to get to the admin page.

example: 

![image](https://user-images.githubusercontent.com/53701945/224283155-a0f4f4e5-05eb-48f0-85ce-38f0bdc6acbe.png)

**Reply by yashkusumkar02** [reply]
Hello Sir , I also provided this path to my project and also added this key to ganache custom network but it is showing me like this and admin section not showing


https://github.com/arlbibek/dVoting/assets/87336351/3103acf8-9a6e-4018-bb85-41837ef0b6ab
